.github/workflows/roll-next.yml

```diff
@@ -29,7 +30,7 @@ jobs:
       - name: Prepare branch
         id: prepare-branch
         run: |
-          BRANCH_NAME="roll/$(date +"%d-%m-%y")"
```

Comment:
Is this related?

Reply:
Yes, to make sure these two automated workflows don't clash with each-other. Since both use the same branch name format.